vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -275,6 +275,11 @@ fire_and_forget WinRTWebSocketResource::PerformClose()
       {
         message = Utf16ToUtf8(e.message());
       }
+      // BSTR's allocated memory must be explicitly released.
+      // See https://docs.microsoft.com/en-us/cpp/atl-mfc-shared/allocating-and-releasing-memory-for-a-bstr
+      SysFreeString(description);
```

Comment:
can you use a wrapper that handles the resource allocation/deallocation like _bstr_t?

Reply:
I don't see why not.
I'll add this to the next master PR, so we don't have discrepant code between branches.

Reply:
@JunielKatarn  @Khalef1  see my comment on the previous PR where I recommended which to use.

Reply:
@asklar  I do recall you mentioned the `wil` library, I asked this follow-up question:
![image](https://user-images.githubusercontent.com/4507319/78730823-c920a500-78f2-11ea-8894-2d95e54e9718.png)

Reply:
So, wouldn't it be simpler to use `b_str_t` as @Khalef1 suggests, so we don't add another external source project dependency just for this specific item?

Reply:
@asklar I don't think we should add `wil` to this repo when we alredy have the Office equivalent of Mso in use. We internally have something called `Mso::BStrHolder`, `Mso::HandleHolder` etc that will be open sourced as needed.